cfpb/consumerfinance.gov
# Set r-o-u-t mapbox token via standard backend process

When this was moved to cf.gov from hmda, the way the mapbox token was set was never updated. Checking in this token to source control (while not super harmful) isn't a best practice and makes abuse easier. This uses the token that's set in `settings`, passes it through to the jinja `context`, which then adds it to the window object for the frontend app to use.

## Testing
 - Pull and build
 - Make sure you have the mapbox token set in your `.env`: `export MAPBOX_ACCESS_TOKEN=<the token I'm removing>`
 - Visit [rout](http://localhost:8000/rural-or-underserved-tool/) and make sure the map still works (click show map after entering an address).
